defaults/main.yml

```diff
@@ -9,108 +9,221 @@
                              "/public" }}'
 
+                                                                   # ]]]
+# .. envvar:: roundcube__src [[[
+#
+# Base path for git bare repository with Roundcube source
+roundcube__src: '{{ (ansible_local.root.src
+                      if (ansible_local|d() and ansible_local.root|d() and
+                          ansible_local.root.src|d())
+                      else "/usr/local/src") + "/" + roundcube__user }}'
+
+                                                                   # ]]]
+# .. envvar:: roundcube__www [[[
+#
+# Base web root directory for Roundcube website
+roundcube__www: '{{ (ansible_local.nginx.www
+                    if (ansible_local|d() and
+                        ansible_local.nginx|d())
+                    else "/srv/www" ) + "/" + roundcube__user }}'
+
+                                                                   # ]]]
+# .. envvar:: roundcube__webserver_user [[[
+#
+# Roundcube webserver user (needs read-only access to the website code)
+roundcube__webserver_user: '{{ ansible_local.nginx.user
+                              if (ansible_local|d() and
+                                  ansible_local.nginx|d() and
+                                  ansible_local.nginx.user|d())
+                              else "www-data" }}'
 
+                                                                   # ]]]
+                                                                   # ]]]
 # .. Database configuration [[[1
 #
 # --------------------------
 #   Database configuration
 # --------------------------
 
-# .. envvar:: roundcube__database_user
+# .. envvar:: roundcube__database [[[
 #
-# Roundcube database user account
-roundcube__database_user: 'roundcube'
-
+# Database definition to use from the :envvar:`roundcube__database_map`.
+roundcube__database: 'sqlite-default'
 
-# .. envvar:: roundcube__database_password
+                                                                   # ]]]
+# .. envvar:: roundcube__database_user [[[
 #
-# Roundcube database password
-roundcube__database_password: "{{ lookup('password', secret + '/credentials/' + ansible_fqdn + '/roundcube/' + roundcube__database + '/' + roundcube__database_user + '/password length=30') }}"
-
+# Database user account to use for Roundcube.
+roundcube__database_user: 'roundcube'
 
-# .. envvar:: roundcube__database
+                                                                   # ]]]
+# .. envvar:: roundcube__database_password_path [[[
 #
-# Database definition to use from the :envvar:`roundcube__database_map`
-roundcube__database: 'sqlite-default'
+# Path to the database password file.
+roundcube__database_password_path: '{{ secret + "/credentials/" + ansible_fqdn
+                                       + "/roundcube/" + roundcube__database
+                                       + "/" + roundcube__database_user + "/password" }}'
+                                                                   # ]]]
```

Comment:
If the database user account is handled by `debops.mariadb` role, the password stored in the `secret/` directory is in a different directory. Did you test this PR with a MariaDB database?